Cargo.toml

```diff
@@ -22,7 +22,7 @@ debug-logs = []
 
 [dependencies]
 breakpad-handler = { version = "0.1.0", path = "./breakpad-handler" }
-sentry-core = { version = "0.28", features = ["client"] }
+sentry-core = { version = ">=0.29", features = ["client"] }
```

Comment:
Not a good idea. See for example what happened with https://github.com/gwihlidal/vk-mem-rs/issues/54.

Reply:
I'll fix it if it breaks in the future, but the constant bumping is incredibly tedious especially when there are rarely breaking changes.

Reply:
We'll let you know when it breaks then :wink: